docs/get-started/getting-started-gcp.md

```diff
@@ -89,8 +93,11 @@ If you'd like to further accelerate init time to 4-5 minutes, create a custom Da
 To use notebooks with a Dataproc cluster, click on the cluster name under the Dataproc cluster tab
 and navigate to the "Web Interfaces" tab.  Under "Web Interfaces", click on the JupyterLab or
 Jupyter link to start to use sample 
-[Mortgage ETL on GPU Jupyter Notebook](../demo/GCP/Mortgage-ETL-GPU.ipynb) to process full 17 years 
-[Mortgage data](https://docs.rapids.ai/datasets/mortgage-data).
+[Mortgage ETL on GPU Jupyter Notebook](../demo/GCP/Mortgage-ETL-GPU.ipynb).
```

Comment:
how does hte ipynb file get to the notebook, is there an easy way to open it from the notebook directly or do you have to download and then reupload?

Reply:
As per my test, only upload/download option. So I just added some words to let user firstly download and upload it in jupyter.